api/anilist/src/main/graphql/UserQuery.graphql

```diff
@@ -12,6 +12,7 @@ query UserQuery(
         search: $search
         sort: $sort
     ) {
+        # TODO: Use fragments.
```

Comment:
What's a fragment 👀

Reply:
We can essentially reuse the fields below in `UserQuery` and `ViewerQuery`